.github/workflows/test.yml

```diff
@@ -298,7 +298,7 @@ jobs:
             "submit-textarea-fields-with-special-characters/**/*.spec.js",
             "task-list/**/*.spec.js",
             "your-business/alternative-trading-address/**/*.spec.js",
-            "your-business/change-your-answers/**/*.spec.js",
```

Comment:
they should still all run with this?

Reply:
Whoops you're right, I thought this was failing in the main branch but it's not. Reverted 👍